frontend/src/lib/components/VersionChecker/versionCheckerLogic.ts

```diff
@@ -44,9 +52,9 @@ export const versionCheckerLogic = kea<versionCheckerLogicType>([
                         kind: NodeKind.HogQLQuery,
                         query: `SELECT properties.$lib_version AS lib_version, max(timestamp) AS latest_timestamp, count(lib_version) as count
                                 FROM events
-                                WHERE timestamp >= now() - INTERVAL 1 DAY 
+                                WHERE timestamp >= now() - INTERVAL ${props.numberOfDays || 1} DAY 
                                 AND timestamp <= now()
-                                AND properties.$lib = 'web'
+                                AND properties.$lib = '${props.lib || 'web'}'
```

Comment:
Something to consider, https://github.com/PostHog/posthog-flutter/issues/59
Flutter events are reported as Android and iOS events since they use the Native SDKs.
In this case, we'd show the banner to Flutter users but we don't support Session recording for Flutter yet.

Reply:
🤔 this smacks of the sam problem as the mobile dashboard filtering...

What's the canonical way to detect this? (and can we add one if there isn't one 😊)

Reply:
AFAIK there's no way since the events are the same, just a different UI platform but the same OS.
Adressing https://github.com/PostHog/posthog-flutter/issues/59 would work but only when people upgrade their SDKs.
Since the focus for now is to get more beta users, we can keep this as it is, and mention in the docs that Flutter isn't supported.

Reply:
regarding:
```
                                AND properties.$lib = '${props.lib || 'web'}'
```

We have the `hogql` helper to sanitize strings properly:

https://github.com/PostHog/posthog/blob/d448593c3b6cb24d9d1362a8b17c5072a86de7a4/frontend/src/queries/utils.ts#L375-L381


```
hogql`SELECT * FROM events WHERE properties.foo = ${'bar'} AND properties.baz = ${3}`
```